scripts/uosc/lib/std.lua

```diff
@@ -37,6 +37,20 @@ function split(str, pattern)
 	return list
 end
 
+---Get index of the last appearance of `sub` in `str`
+---@param str string
+---@param sub string
+---@return integer|nil
+function string.last_index_of(str, sub)
```

Comment:
Don't extend `string`, this might cause issues. Make it `string_last_index_of*()`.

Reply:
Nvm, I'll do it alter, I want to refactor the directory structure to support mpv 0.32 now.